File: week4/examples/phonebook.c
#include <stdio.h>
#include <cs50.h>


int main(void)
{
    // Open file
    FILE *file = fopen("phonebook.csv", "a");

    // Print (write) to file opened
    char *name = get_string("Name: ");
    char *number = get_string("Number: ");

    fprintf(file, "%s,%s\n", name, number);
    fclose(file);

    return 0;
}

File: week6/examples/phonebook.py
# handling file
import csv

from cs50 import get_string


name = get_string("Name: ")
number = get_string("Number: ")

# If use `with`, the file.close() runs automatically
with open("phonebook.csv", "a") as file:
    writer = csv.writer(file)
    writer.writerow((name, number))

# file = open("phonebook.csv", "a")
# writer = csv.writer(file)
# writer.writerow((name, number))
# file.close()
